lib/search.js

```diff
@@ -15,6 +15,8 @@ const PAGE = 1 // page_number is 1-based index
 const MAX_PAGE_ALLOWED = 34
 
 async function search (q, auth) {
+  const octokit = auth ? Octokit({ auth: auth }) : Octokit()
```

Comment:
Yeah that should work 👍 

```suggestion
   const octokit = auth ? Octokit({ auth }) : Octokit()
```